app/models/smart_proxy.rb

```diff
@@ -90,6 +90,25 @@ def has_feature?(feature)
     self.features.any? { |proxy_feature| proxy_feature.name == feature }
   end
 
+  def version
+    begin
+      version = { :status => N_('OK'), :message => ProxyAPI::Version.new(:url => url).version }
```

Comment:
I'd suggest putting `version` in a key called version rather than message - message can always be a human-readable message like the LDAP test.